mesonbuild/dependencies/pkgconfig.py

```diff
@@ -89,12 +90,6 @@ def list_all(self) -> T.List[str]:
 class PkgConfigCLI(PkgConfigInterface):
     '''pkg-config CLI implementation'''
 
-    # We cache all pkg-config subprocess invocations to avoid redundant calls
-    pkgbin_cache: T.Dict[
```

Comment:
I have not checked, just curious... does either one or both approaches retain the cache as part of the pickled data which we reuse when reconfiguring?